pepper-apis/src/test/java/org/broadinstitute/ddp/client/GoogleRecaptchaClientTest.java

```diff
@@ -0,0 +1,17 @@
+package org.broadinstitute.ddp.client;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+public class GoogleRecaptchaClientTest {
+    @Test
+    public void testReachingGoogle() {
+        GoogleRecaptchaVerifyResponse verifyResponse = new GoogleRecaptchaVerifyClient("nothing").verifyRecaptchaResponse("1234565");
+        assertNotNull(verifyResponse);
+        assertFalse(verifyResponse.isSuccess());
+        assertTrue(verifyResponse.getErrorCodes().size() > 0 );
+    }
```

Comment:
This really just tests that we can get to Google and the deserialization is not totally screwed up. Not sure how to do a "real" comprehensive test, but I consider this to be sufficient. Was helpful getting the http call and JSON conversion straight.